.rubocop/strict.yml

```diff
@@ -0,0 +1,23 @@
+# Inherit from TODO here to make sure we enforce the rules below (and TODO is ignored)
+inherit_from:
+  - ../.rubocop_todo.yml
```

Comment:
I don't understand this. `rubocop_todo.yml` is already inherited before this file in the main rubocop file, so what's the difference?

Reply:
I think this is just overly cautious for an edge case where you are running rubocop not via that top level config but you are using the strict approach and want to be sure it loads after the todo.

In our usage, where I suspect 99% of runs are just via `rubocop` with no specials args/configs and it does go through the main config, it would be safe to drop.

Don't feel strongly.

Reply:
Yeah, I think I would drop it. Otherwise this looks fine.

Reply:
dropped